packages/ckeditor5-image/tests/imageinsert/imageinsertui.js

```diff
@@ -74,10 +74,26 @@ describe( 'ImageInsertUI', () => {
 			return editor.destroy();
 		} );
 
-		it( 'should register the "insertImage" dropdown', () => {
-			const dropdown = editor.ui.componentFactory.create( 'insertImage' );
+		describe( 'should register the "insertImage" dropdown', () => {
+			it( 'button with proper properties set', () => {
+				expect( dropdown ).to.be.instanceOf( DropdownView );
```

Comment:
What's the reason for checking dropdown's variable type here?

I assume that's a leftover from the removed unit test - however it was doing a different assessment from what you're doing here. It was checking that a proper type is returned by `insertImage` component - while tests that you wanted to added are only interested in checking button or its arrow properties.

Original test should be kept and these tests should assume that the `dropdown` has a proper type.